test/fixtures/poro.rb

```diff
@@ -38,9 +38,16 @@ class ProfilePreviewSerializer < ActiveModel::Serializer
 
 Post     = Class.new(Model)
 Like     = Class.new(Model)
-Author   = Class.new(Model)
+Author   = Class.new(Model) do
+  # To confirm cache_key is set using updated_at and cache_key option passed to cache
+  undef_method :cache_key
+end
 Bio      = Class.new(Model)
 Blog     = Class.new(Model)
+Article  = Class.new(Model) do
+  # To confirm error is raised when cache_key is not set and cache_key option not passed to cache
+  undef_method :cache_key
+end
```

Comment:
Great, I'll move my new objects.  I'm a fan of each test being responsible for it's own objects, but I'm a bigger fan of consistency.  Wanted to double check.